src/drone_controller/exception/exceptions.py

```diff
@@ -0,0 +1,15 @@
+class DroneControllerError(Exception):
+    """
+    Base class for exception in this module.
+    """
+
+
+class UserInputError(DroneControllerError):
+    """
+    Incorrect user input.
+    """
+
+    def __init__(self, expression, message):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Missing docstring in __init__](https://app.codacy.com/gh/treeandsea/DroneController/pullRequest?prid=5205179)